### PR TITLE
[Data][Matrix] Migrate to EventTree from Commons

### PR DESCRIPTION
**Pull request by Herklos**
Requires https://github.com/Drakkar-Software/OctoBot-Commons/pull/4

### CODE REVIEW

**Comment by Herklos** [comment]
Sorry for the huge number of line changes but it mainly due to the test_matrix file move and the removal of the legacy methods.

File: tests/data/test_matrix.py [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+    btc_node = matrix.matrix.get_or_create_node(get_tentacle_value_path(symbol="BTC"), starting_node=created_node)
+    eth_node = matrix.matrix.get_or_create_node(get_tentacle_value_path(symbol="ETH"), starting_node=created_node)
+    btc_node_2 = matrix.matrix.get_or_create_node(get_tentacle_value_path(symbol="BTC"), starting_node=created_node_2)
+    assert matrix.get_tentacles_value_nodes(tentacle_nodes=[created_node, created_node_2],
+                                            symbol="BTC") == [btc_node, btc_node_2]
+    assert matrix.get_tentacles_value_nodes(tentacle_nodes=[created_node, created_node_2],
+                                            symbol="ETH") == [eth_node]
+
+# @pytest.mark.asyncio
+# async def test_get_tentacles_value_nodes_with_time_frame():
```
Comment (GuillaumeDSM) [inline tests/data/test_matrix.py]: WIP ?

Comment (Herklos) [reply tests/data/test_matrix.py]: Yes it was, sorry, I fixed it.

File: demo.py [hunk 2]
```diff
@@ -60,9 +59,9 @@
     await initialize_evaluators(config)
     await create_all_type_evaluators(config, "test", ["BTC/USDT"], [TimeFrames.ONE_HOUR])
 
-    # await get_chan(MATRIX_CHANNEL).get_internal_producer().send(evaluator_name="test",
-    #                                                                            evaluator_type="test",
-    #                                                                            eval_note=1)
+    await get_chan(MATRIX_CHANNEL).get_internal_producer().send(evaluator_name="test",
+                                                                               evaluator_type="test",
+                                                                               eval_note=1)
```
[the remark this answers is not in the document]
Comment (Herklos) [reply demo.py]: Yes, done ;)

**Pull request merged**